src/browser/atom-portable.coffee

```diff
@@ -1,6 +1,6 @@
 fs = require 'fs-plus'
 path = require 'path'
-ipc = require 'ipc'
+ipc = require 'ipc-main'
```

Comment:
I'm thoroughly confused by this right now.  Tried both `ipc-main` and `ipc-renderer`, but both throw "Cannot find module" errors.

(It's probably also why core specs finish in 13 secs and fail)